src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -0,0 +1,92 @@
+# Managing Guice distributed James
+
+This guides aims to be an entry-point to the James documentation for user managing a distributed Guice James server.
+
+It includes:
+
+ - Simple architecture explanations
+ - Propose some diagnostic for some common issues
+ - Present procedures that can be set up to address these issues
+
+In order to not duplicate information, existing documentation will be linked.
+
+Please note that this product is under active development, should be considered experimental and thus is addressed to 
+advanced users.
+
+## Table of content
+
+ - [Overall architecture](#overall-architecture)
+ - [Basic Monitoring](#basic-monitoring)
+
+## Overall architecture
+
+Guice distributed James server intends to provide a horizontally scalable email server.
+
+In order to achieve this goal, this product leverages the following technologies:
+
+ - **Cassandra** for meta-data storage
+ - **ObjectStorage** (S3) for binary content storage
```

Comment:
Is S3 the case only? I thought we support Swift & S3

Reply:
I may anticipate that one https://github.com/linagora/james-project/pull/3099